api/envoy/config/listener/v3/udp_listener_config.proto

```diff
@@ -33,8 +33,18 @@ message UdpListenerConfig {
 
   // Configuration for QUIC protocol. If empty, QUIC will not be enabled on this listener. Set
   // to the default object to enable QUIC without modifying any additional options.
-  // [#not-implemented-hide:]
-  // [#comment:Unhide when QUIC alpha is announced with other docs.]
+  //
+  // .. note::
+  //   QUIC support is currently alpha and should be used with caution.
+  //
+  //   For known outstanding issues before Envoy QUIC support is GA, issues
+  //   are tracked `here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_.
+  //   For example QUIC does not currently support in-place filter chain updates, so users
+  //   requiring dynamic config reload for QUIC should wait until
+  //   `#13115 <https://github.com/envoyproxy/envoy/issues/13115>_` has been addressed.
+  //
+  //   For general feature requests beyond production readiness, you can track
+  //   updates `here <https://github.com/envoyproxy/envoy/labels/area%2Fquic>`.
```

Comment:
It seems that this link, and the one above, are not rendered properly.
See example here: https://storage.googleapis.com/envoy-pr/e10ad5f/docs/api-v3/config/listener/v3/udp_listener_config.proto.html?highlight=quicprotocoloptions